Jenkinsfile

```diff
@@ -0,0 +1,109 @@
+/**
+* IOS Jenkinsfile
+*/
+
+//     in RHMAP's case, following parameters are sent by RHMAP to Jenkins job.
+//     this means, Jenkins job has to be a parametrized build with those parameter.
+CODE_SIGN_PROFILE_ID = params?.BUILD_CREDENTIAL_ID?.trim()                 // e.g. "redhat-dist-dp"
+BUILD_CONFIG = params?.BUILD_CONFIG?.trim()                                // e.g. "Debug" or "Release"
+
+//     if you would like to hardcode these, do it this way
+//CODE_SIGN_PROFILE_ID = "redhat-dist-dp"
+//BUILD_CONFIG = "Debug"
+
+// sample values commented below are for https://github.com/feedhenry-templates/helloworld-ios-swift
+/* ------------- use these to hardcode things in Jenkinsfile ---------------- */
+PROJECT_NAME = "helloworld-ios-swift"
+INFO_PLIST = "helloworld-ios-swift/helloworld-ios-swift-Info.plist"
```

Comment:
sholuld be `helloworld-ios-app/helloworld-ios-app-Info.plist` as it should point to this file: https://github.com/feedhenry-templates/helloworld-ios-swift/blob/master/helloworld-ios-app/helloworld-ios-app-Info.plist